t/t2230-job-info-list.t

```diff
@@ -748,6 +748,17 @@ test_expect_success HAVE_JQ 'flux job list outputs exceptions correctly (excepti
         echo $obj | jq .exception_note | grep "No such file or directory"
 '
 
+# expiration time
+
+test_expect_success HAVE_JQ 'flux job list outputs expiration time when set' '
+	jobid=$(flux mini submit -t 30 sleep 1000) &&
+	fj_wait_event $jobid start &&
+	flux job list | grep $jobid > expiration.json &&
+	test_debug "cat expiration.json" &&
+	jq -e ".expiration > now" < expiration.json &&
+	flux job cancel $jobid
+'
+
```

Comment:
wasn't sure where to leave this comment, but down below in this file there's also a test that the `list-attrs` command is listing all attributes correctly.